posts/pos2020_zhTW.md

```diff
@@ -9,81 +9,81 @@ _感谢Chih-Cheng Liang和Hsiao-wei Wang翻译这个文章_
 
 ## 權益證明在同樣成本下可提供更高的安全性
 
-理解這點最簡單的方式就是把權益證明和工作量證明擺在一起看，假設每天有 $1 的區塊獎勵，攻擊此網路所需的成本是多少。
+理解這點最簡單的方式，就是把權益證明和工作量證明擺在一起看，**假設每天有 $1 的區塊獎勵，攻擊此網路所需的成本是多少。**
 
 ### 基於 GPU 的工作量證明
 
-你可以便宜租到 GPU，所以攻擊網路的成本就只是租到足夠的 GPU 算力去壓過現有的礦工。在區塊獎勵每花的 $1，現有的礦工應該花費將近 $1 的成本（如果他們花費更多，礦工就會因為無利可圖而退出；如果花費比這個數字少，新礦工就會加入來獲取利潤）。因此，攻擊網路只需要暫時花費比 $1/天 多一點點，而且可能只需要持續幾個小時。
+你可以便宜租到 GPU，所以攻擊網路的成本就等於是租到夠強 GPU 的成本，只要能夠壓過現有礦工 GPU 的算力即可。現有的礦工會花費將近 $1 的成本來賺取 $1 的區塊獎勵（如果他們花費更多，礦工就會因為無利可圖而退出；如果花費比這個數字少，新礦工就會加入來壓縮現有礦工的利潤空間）。因此，攻擊網路只需要暫時花費比 $1/天 多一點點，這還可能只要幾個小時而已。
 
-**總攻擊成本： ~$0.26 （假設攻擊 6 小時），且因為攻擊者可以收到區塊獎勵，所以這個數字還有可能壓到零**
+**總攻擊成本： ~$0.26 （假設攻擊 6 小時）**，且因為攻擊者有機會收到區塊獎勵，所以這個數字可能壓到零
 
 ### 基於 ASIC 的工作量證明
 
-ASICs 其實是資本成本：當買進 ASIC 時你預期大概可以用兩年，因為它會慢慢耗損或是因為更新更好的硬體出來而過時。如果一個鏈被 51% 攻擊了，社群大概會用更換 PoW 演算法應對，而這時你的 ASIC 就會失去價值。平均而言，用挖礦大概是大約 1/3 的經常性成本與 2/3 的資本成本（詳見 [這裡](https://eth.wiki/concepts/proof-of-stake-faqs#what-about-capital-lockup-costs)）。因此，每天每花費 $1 在區塊獎勵上，礦工每天會花 ~$0.33 在電力與維護上，並花費 ~$0.67 在他們的 ASIC 上。假設 ASIC 可以用 ~2 年，礦工會需要為那單位的 ASIC 硬體花費 $486.67 。
+ASICs 其實是資本成本：當買進 ASIC 時你預期大概可以用兩年，因為它會慢慢耗損，或是因為更新更好的硬體出來而過時。如果一個鏈被 51% 攻擊了，社群大概會用更換 PoW 演算法來應對，這時，你的 ASIC 就會失去價值。平均而言，挖礦所費的成本分佈是 1/3 的經常性成本與 2/3 的資本成本（詳見 [這裡](https://eth.wiki/concepts/proof-of-stake-faqs#what-about-capital-lockup-costs)）。因此，為了每天要賺到 $1 的區塊獎勵，礦工得花 ~$0.33 在電力與維護上，花費 ~$0.67 在 ASIC 上。假設 ASIC 可以用 ~2 年，礦工會需要為一單位的 ASIC 硬體花費 $486.67 。
 
 > 譯註：$486.67 = 365 天 x 2 年 x $0.67 資本成本
 
 **總攻擊成本：$486.67 （ASICs） + $0.08（電力與維護）= $486.75**
 
 > 譯注：這邊電力與維護成本也是假設攻擊 6 小時
 
-話雖如此，值得注意的是 ASICs （相較 GPU）帶來較高的安全性是以中心化的高代價換來的，因此[加入 ASIC 挖礦的門檻也非常高](https://blog.ethereum.org/2014/06/19/mining/)。
+值得注意的是，雖然 ASICs以高度的中心化此一昂貴的代價換來了較高的安全性，這同時也讓[加入 ASIC 挖礦的門檻變得非常高](https://blog.ethereum.org/2014/06/19/mining/)。
 
 ### 權益證明 Proof of stake
 
-權益證明的成本幾乎是百分百的資本成本（抵押的幣）；唯一的營運成本是運行節點的成本。這樣人們會願意為每天每 $1 的區塊獎勵鎖住多少的資金呢？不像 ASIC，抵押的幣不會折舊，而且當你不想抵押了你還可以在一段短時間內取回你的幣。因此，參與者應該會願意為同樣程度的獎勵付出比 ASIC 的情況更高的資本成本。
+權益證明的成本幾乎全是資本成本（抵押的幣）；唯一的營運成本花在運行節點上。猜猜人們願意為每天 $1 的區塊獎勵鎖住多少的資金呢？**不像 ASIC，抵押的幣不會折舊，而且當你不想抵押了，你幾乎馬上就可以取回你的幣。因此，在獎勵相同的情況下，參與者會願意付出比 ASIC 的情況更多的資本成本。**
 
-讓我們假設 ~15% 的報酬率足夠吸引人們抵押（這是 eth2 的期望報酬率）。因此每天 $1 的區塊獎勵會吸引相當於 6.667 年報酬的抵押，或換算為金額為 $2,433 。節點消耗的硬體與電力成本很小，每一千元的電腦可以抵押上百上千元，而且每月 ~$100 的電力與網路費也算足夠。但保守的說，我們假設這些經常性成本是抵押總成本的 ~10%。所以我們只有每天 $0.90 的區塊獎勵對應到資本成本，因此我們還要把上面的數字減少 ~10%。
+讓我們假設 ~15% 的報酬率足夠吸引人們抵押（這是 eth2 的期望報酬率）。因此每天 $1 的區塊獎勵會吸引相當於 6.667 年報酬的抵押，可換算成 $2,433 。節點消耗的硬體與電力成本很小，一台千元美金等級的電腦就可以抵押非常大量的幣，而且每月只要花大概 ~$100 左右的電力與網路費就綽綽有餘。我們可以保守的假設上述這些經常性成本為抵押總成本的 10% ，所以攻擊時的資本成本會讓我們每天能拿到的區塊獎勵剩下 $0.90 ，這讓上面算出來的抵押數字也必須跟著少 10% 。
```

Comment:
> 所以攻擊時的資本成本會讓我們每天能拿到的區塊獎勵剩下 $0.90

The source text is "so we only have $0.90 per day of rewards that end up corresponding to capital costs." 

My understanding is that people are willing to deposit $0.90/ 15% to earn $1 of the block reward, so that attacker needs to spend $0.90/ 15% too. It looks like the relation between the block reward and the attacker's cost is not fully captured in the translated text.

Reply:
我們可以保守的假設上述這些經常性成本為抵押總成本的 10% ，這代表在你每天能賺到的區塊獎勵中，需要扣掉 10% 的經常性成本，剩下的 $0.90 才是（攻擊時）需要花費的資本成本。

大概是這樣，Buterin原本句子太隨意了，當翻譯有時候在你逐字翻譯會讓讀者不知道在看三小的時候，需要重寫句子

這個改動就屬於重寫句子的一種，你看看如何